scripts/smp-server-droplet/scripts/03-prep.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# retrieve latest release info and download smp-server executable
+echo "downloading SMP server release $release_tag"
+curl -s https://api.github.com/repos/simplex-chat/simplexmq/releases/tags/{$release_tag} \
+| jq -r '.assets[].browser_download_url | select(test("smp-server-ubuntu-20_04-x86-64"))' \
+| tr -d \" \
+| wget -qi -
+
+echo "preparing for SMP server initiaization on first login"
+# move smp-server executable to /opt/simplex/bin
```

Comment:
don't think comment is needed here, commands below are self-explanatory

Reply:
removed